src/main/java/org/ibp/api/ibpworkbench/rest/BreedingView.java

```diff
@@ -43,31 +48,36 @@
 	@Autowired
 	private BreedingViewService breedingViewService;
 
+	@Resource
+	private DesignLicenseService designLicenseService;
+
 	@ApiOperation(value = "Save the Single-Site Analysis CSV output file with heritability", notes = "", response = DataResponse.class)
 	@RequestMapping(value = "{cropName}/ssa/save_result_summary", method = RequestMethod.GET, produces = {MediaType.APPLICATION_XML_VALUE})
 	@ResponseBody
 	public DataResponse saveSsaResultSummary(
```

Comment:
This resource and class does not follow any REST standard at all 👎 We should review it later, but I left a comment to create a different resource for the BVLicense